package.json

```
@@ -41,7 +41,8 @@
     "docs"
   ],
   "scripts": {
-    "build:node": "npx esbuild@0.14.38 index-fetch.js --bundle --platform=node --outfile=undici-fetch.js",
+    "build:node": "npx esbuild@0.15.15 index-fetch.js --bundle --external:readable-stream --platform=node --outfile=undici-fetch.js",
+    "postbuild:node": "sed -i 's/require(\"readable-stream\")/require(\"stream\")/g' undici-fetch.js",
```

```suggestion
    "build:node": "npx esbuild@0.15.16 index-fetch.js --bundle --alias:readable-stream=stream --platform=node --outfile=undici-fetch.js",
```

esbuild just added a flag for this: https://github.com/evanw/esbuild/releases/tag/v0.15.16